Description: skip test because bogus identify -verbose output
Bug: https://bugs.debian.org/966436
Author: Cédric Boutillier <boutil@debian.org>
Last-Update: 2020-07-29

--- a/spec/lib/mini_magick/image_spec.rb
+++ b/spec/lib/mini_magick/image_spec.rb
@@ -471,7 +471,7 @@
         context "when verbose information includes a clipping path", skip_cli: :graphicsmagick do
           subject { described_class.new(image_path(:clipping_path)) }
 
-          it "does not hang when parsing verbose data" do
+          xit "does not hang when parsing verbose data" do
             # Retrieving .details should happen very quickly but as of v4.3.6
             # will hang indefinitely without the timeout
             Timeout::timeout(10) do
